processes/README.MD

```diff
@@ -0,0 +1,18 @@
+# Documented Processes
```

Comment:
how happy is the Markdown toolchain going to be with uppercase file extensions?

Reply:
Not sure which toolchain - or which extension - you mean. Do you have a specific downstream tool in mind?

Reply:
No just TOLing.